src/assets/styles/style.scss

```diff
@@ -0,0 +1,10 @@
+body {
+  height: 100vh;
+  background: rgb(18, 50, 95);
+  background: linear-gradient(180deg, #855798 6%, #cca097 75%);
+  background-repeat: no-repeat;
+}
+
+.form-control::placeholder {
+  color: white !important;
```

Comment:
- [ ] Using !important is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. When two conflicting declarations with the !important rule are applied to the same element, the declaration with a greater specificity will be applied

Reply:
Sorry I used important there because it's used to overwrite bootstrap default style

Reply:
I think you can create a new CSS class for this color.